libs/LedKit/source/LedKit.cpp

```diff
@@ -4,9 +4,21 @@
 
 #include "LedKit.h"
 
+#include "rtos/ThisThread.h"
+
 using namespace leka;
 using namespace std::chrono;
 
+void LedKit::initializeLeds()
```

Comment:
tu peux l'appeler juste init ou initialize

```suggestion
void LedKit::init()
```